beta-src/src/data/map/land/LandTerritoriesMapData.ts

```diff
@@ -360,6 +360,66 @@ export const BULGARIA: TerritoryMapData = new TerritoryMapDataGenerator({
   path: "M208.68 90.0601C207.08 87.4601 205.42 84.5801 205.88 81.5601C206.55 77.1901 209.54 72.35 211.23 68.26C213.96 61.61 213.43 54.14 213.3 46.95C213.152 46.0509 213.253 45.1287 213.592 44.2827C213.93 43.4367 214.493 42.6991 215.22 42.1499C219.469 38.1763 224.023 34.542 228.84 31.28C230.42 30.21 232.12 29.07 232.84 27.28C233.66 25.28 232.95 22.96 232.54 20.8C232.273 19.3812 232.119 17.9433 232.08 16.5C228.405 15.3476 224.647 14.475 220.84 13.8899C196.3 9.73989 170.51 8.04001 147.02 16.26C141.38 18.26 135.92 20.77 130.12 22.26C123.68 23.87 116.99 24.1001 110.36 24.3301C84.1601 25.2201 57.7099 26.1001 31.9299 21.3301C27.7399 20.5601 23.35 19.5201 20.28 16.5801C18.28 14.6301 16.9901 12.0199 15.1101 9.92993C11.4746 6.04441 7.03921 2.99426 2.11006 0.98999C2.66797 9.77123 2.33292 18.5865 1.11006 27.3C0.674627 29.6742 0.486895 32.0871 0.549999 34.5C0.899999 39.39 3.54992 43.7799 6.21992 47.9199C10.0599 53.9199 14.1699 60.2501 14.1699 67.3501C14.1699 74.4501 9.9799 80.9401 5.9499 86.8701C6.2399 91.7901 6.32993 97.47 7.72993 102.2C10.5699 111.8 17.22 120.71 16.31 130.68C15.9644 134.015 14.7798 137.209 12.8669 139.963C10.9539 142.717 8.37494 144.942 5.37007 146.43C2.57007 147.78 3.64995 152.57 5.49995 155.07C7.34995 157.57 10.1598 159.07 12.8898 160.59C15.3898 161.94 17.99 163.32 20.83 163.48C23.3461 163.465 25.8352 162.963 28.1599 162L68.9799 148.07C73.0799 146.68 77.4399 145.26 81.6699 146.21C85.8999 147.16 89.48 150.58 91.52 154.52C93.56 158.46 94.3699 162.9 94.9399 167.3C95.6597 172.912 96.0505 178.562 96.1101 184.22C98.5401 182.5 101.74 182.15 104.75 181.94C111.269 181.394 117.82 181.317 124.35 181.71C126.48 181.86 128.6 182.09 130.72 182.33L140.47 183.39L145.75 183.97C147.06 184.12 149.03 183.97 150.15 184.76C150.247 184.832 150.337 184.912 150.42 185L150.99 184.29C155.42 178.68 159.35 172.5 161.13 165.58C163.13 157.91 162.29 149.85 162.51 141.94C162.73 134.03 164.25 125.59 169.78 119.94C174.72 114.86 182.08 112.94 189.14 112.37C196.2 111.8 203.34 112.54 210.39 111.79C213.854 111.584 217.213 110.526 220.17 108.71C216.337 102.49 212.507 96.2734 208.68 90.0601Z",
 }).territory;
 
+export const BULGARIA_NORTH_COAST: TerritoryMapData =
+  new TerritoryMapDataGenerator({
+    ...new TerritoryDataGenerator(Territory.BULGARIA_NORTH_COAST).territory,
+    arrowReceiver: {
+      x: 125,
+      y: 75,
+    },
+    centerPos: {
+      x: 20,
+      y: 65,
+    },
+    fill: "none",
+    labels: [],
+    unitSlots: [
+      {
+        name: "main",
+        x: 35,
+        y: 95,
+      },
+    ],
+    width: 233,
+    height: 81,
+    x: 3284,
+    y: 2122,
+    texture: {
+      texture: Texture.LAND,
+    },
```

Comment:
Can we remove textures, labels, centers, and unit slots for these since we don't need them.

Reply:
Removed unneeded options